library/user_term.py

```diff
@@ -0,0 +1,56 @@
+import dataclasses
+from textwrap import dedent
+
+
+@dataclasses.dataclass()
+class UserTermState:
+    """Reset machine for next turn"""
+
+    subgrid_select: bool = True
+    space_select: bool = True
+
+    wait_for_ready: bool = True
+    user_confirm: bool = True
+    start_of_turn: bool = True
+    end_of_turn: bool = False
+    save_subgrid: bool = True
+
+    user_select_subgrid: int = 0
+    user_select_space: int = 0
+
+
+def starting_user_section() -> str:
+    """Starting Terminal"""
+    output = dedent(
+        """
+        ┌─term────────────────────────────┐
+        │                                 │
+        │      shall we play a game?      │
+        │             (y/n?)              │
+        │                                 │
+        └(Enter to confirm)───('q' to esc)┘
+    """
+    ).strip()
+
+    return output
+
+
+def update_user_section(updated_info: list[str]) -> str:
+    """To add your information to the terminal you have 3 lines with 31 spaces"""
+    for i in updated_info:
+        if len(i) > 31:
+            raise NameError("Terminal contents too big")

```

Comment:
Should this be a `ValueError` instead? Or another error, idk.

Reply:
I got rid of it. It was more trouble than it was worth.
I figured it was unlikely the user would cause this to error and if we did see an error it would be in debugging and we would fix it.